akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcOffsetStoreSpec.scala

```diff
@@ -154,8 +154,8 @@ abstract class JdbcOffsetStoreSpec(specConfig: JdbcSpecConfig)
         tryWithResource(stmt.executeQuery()) { resultSet =>
 
           if (resultSet.next()) {
-            val t = resultSet.getTimestamp(6)
-            Instant.ofEpochMilli(t.getTime)
+            val millisSinceEpoch = resultSet.getLong(6)
+            Instant.ofEpochMilli(millisSinceEpoch)
```

Comment:
In tests, I still convert the value to an `Instant` to not rewrite the whole test.